styles.css

```diff
@@ -1,10 +1,23 @@
 
 /* To add some space below the navigation bar */
 
+
  body {
-  margin-top: 100px;
+  margin-top: 30px;
+  font-family: "Open Sans","Helvetica Neue",Helvetica,Arial,sans-serif !important;
 }
 
+/*  Increase page width to make more use of the screen on both sides*/
+
+body .main-container {
+  max-width: 1140px !important;
+  width: 1140px !important;
+}
+body {
+  max-width: 1140px !important;
+  margin-left: auto;
+  margin-right: auto;
+}
 
 /* add some color - if you want color behind a major point on your website */
 
```

Comment:
we can change the banner color below to something that would fit the aesthetic more. you can use a hex color for more specificity if you like: [https://g.co/kgs/9ufQBw](https://g.co/kgs/9ufQBw)

Reply:
Agreed, this requires change.